OpenRA.Mods.Common/Traits/Render/RenderShroudCircle.cs

```diff
@@ -23,8 +23,14 @@ class RenderShroudCircleInfo : TraitInfo, IPlaceBuildingDecorationInfo
 		[Desc("Color of the circle.")]
 		public readonly Color Color = Color.FromArgb(128, Color.Cyan);
 
+		[Desc("Range circle line width.")]
+		public readonly float Width = 1;
+
 		[Desc("Contrast color of the circle.")]
```

Comment:
contrast &rarr; border, and ideally reword the description here and on `Color` to the wording used in the other traits.